src/app/ngrx/reducers/podcast-ranks.reducer.spec.ts

```diff
@@ -38,16 +38,28 @@ describe('Podcast Ranks Reducer', () => {
       key: `${routerParams.podcastId}-${routerParams.group}-${routerParams.interval.key}`,
       podcastId: routerParams.podcastId,
       group: routerParams.group,
+      filter: undefined,
       interval: routerParams.interval,
       downloads: podcastAgentNameDownloads,
-      ranks: podcastAgentNameRanks
+      ranks: podcastAgentNameRanks,
+      loaded: true,
+      loading: false
     });
-    expect(newState.loaded).toBeTruthy();
+  });
+
+  it('should include filter in key for geo subdiv', () => {
```

Comment:
I added filter to the 'key' for geosubdiv entries so multiple country's subdiv entries would be saved on state as loaded. (When collapsed/expanded, the do reload but a spinner is not shown because they already exist in state.)